### PR TITLE
Should say ID Token not Access Token

### PR DESCRIPTION
**Pull request by dogeared**
## Description:
- Docs say Access Token in a place where it should be ID Token

## Type of Pull Request:

- [x] Content (Documentation updates or typo-fixes)
- [ ] Blog Post
- [ ] Functional (CSS changes, JS updates, or layout modifications)

### Resolves:
* #893 

## How Has This Been Tested?
- [x] I have built this locally and verified that it does not break existing functionality.
- [ ] For functional changes, I have tested against Chrome, Firefox, and Safari, and mobile.
- [ ] For functional changes, I have added tests.

## Primary Reviewers:

- *Tag reviewer(s)*

- [Blog Post Google Doc](https://docs.google.com)

### CODE REVIEW

**Comment by rchild-okta** [comment]
Change looks good, but can you follow the steps listed here?
https://github.com/okta/okta.github.io#you-are-making-a-non-blog-change

tldr; these commits go into the `weekly` branch
